src/aktualizr_secondary/aktualizr_secondary_config.cc

```diff
@@ -48,6 +48,30 @@ void AktualizrSecondaryUptaneConfig::writeToStream(std::ostream& out_stream) con
   writeOption(out_stream, key_type, "key_type");
 }
 
+void PackageConfig::writeToStream(std::ostream& out_stream) const {
```

Comment:
These two methods + `std::ostream& operator<<(std::ostream& os, PackageManager pm)` are already implemented in `config.cc` and `aktualizr-secondary` successfully links even when these implementations are removed.

Any specific reason to also define them here?

Reply:
Sorry, I just temporarry copied it.